CHANGELOG.md

```diff
@@ -26,8 +26,9 @@
 
 ## Fixed Issues
 
-- [core] Re-enable the circuit breakers for the destination and xsuaa services.
+- [core] Re-enable the circuit breakers for the destination and XSUAA services.
 - [odata] Fix encoding of filter strings.
+- [core] Add or overwrite the content length header for CSRF fetch requests. As those are always `HEAD` requests, the content length is always set to 0.
```

Comment:
Alternative would be to delete the `content-length` key. I guess setting to 0 might be a bit easier.

Reply:
The CAP colleagues mentioned that some servers need the content length header. That is why they add it.

Reply:
I guess they need it for the real request.
I mean only remove the key for `HEAD` request, which should make sense?

Reply:
But the HEAD request also goes against the same system.